### DISCUSSION TITLE
Capture view on react native SVG

### DISCUSSION THREAD

**Discussion by taduyde** [microsoft/react-native-windows#10376] [Q&A] [answered]
Hello guys, 

as currently, I use this plugin (https://github.com/gre/react-native-view-shot) the capture the view on the windows application, it works well with react native view, 
but when I use react-native-SVG, it can't capture.
Does anybody have a link guide to do the capture view with react native SVG?
Thanks in advance.

**Comment by chrisglein** [comment, marked as answer]
Have you tried asking on the [react-native-svg](https://github.com/react-native-svg/react-native-svg) GitHub? They're going to have more familiarity with any features/compatibility than anyone here.
